script/Arcueid/Day 4/04_00_ARC04_7_1.txt

```diff
@@ -68,7 +68,7 @@ Arcueid falls silent, a serious expression on her face.
 [sha:c9aaa42138d8517b79c7004e171229be16950812]{
 -- Page 644, Offset 7286.
 -- 「知ってるわ。現存する四人の魔法使いの一人だもの。
-"I know of her. She's one of the four extant magicians.
+"I know of her. She's one of the four extant Magicians.
```

Comment:
This is the only case where it should be capitalised imo. 

All the others are just the general societal view of 'magic users'. Kiritsugu introduces himself to Shirou as a magician, as does Touko to Ryougi. Those should not be capitalised.

Reply:
This